ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/block/BlockProcessorCapella.java

```diff
@@ -55,4 +77,76 @@ public BlockProcessorCapella(
         operationValidator,
         schemaDefinitions);
   }
+
+  @Override
+  @CheckReturnValue
+  protected BlockValidationResult validateBlock(
```

Comment:
I think we actually want to override `verifyBlockSignatures` and add in the `verifyBlsToExecutionChanges` at the end of that. Otherwise we're checking the state root matches before verifying the bus execution changes and calculating the state root is a much more expensive validation to perform.